AUTHORS

```diff
@@ -954,6 +954,7 @@ answer newbie questions, and generally made Django that much better:
     Tareque Hossain <http://www.codexn.com>
     Taylor Mitchell <taylor.mitchell@gmail.com>
     Terry Huang <terryh.tp@gmail.com>
+    tell-k <fkk2005@gmail.com>
```

Comment:
Alphabetized, move your entry above the `Terry ...`

Reply:
@felixxm Sorry, I'll fix it right away.